samples/subsys/usb/cdc_acm/Kconfig

```diff
@@ -0,0 +1,4 @@
+config USB_DEVICE_PID
+	default USB_PID_CDC_ACM_SAMPLE
+
+source "Kconfig.zephyr"
```

Comment:
Nit: Your editor doesn't insert newlines at end of files

Reply:
Also git gives warnings
```
diff --git a/samples/subsys/usb/cdc_acm/Kconfig b/samples/subsys/usb/cdc_acm/Kconfig
new file mode 100644
index 0000000000..e5eb86e48e
--- /dev/null
+++ b/samples/subsys/usb/cdc_acm/Kconfig
@@ -0,0 +1,4 @@
+config USB_DEVICE_PID
+       default USB_PID_CDC_ACM_SAMPLE
+
+source "Kconfig.zephyr"
\ No newline at end of file
```